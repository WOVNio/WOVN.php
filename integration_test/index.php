<?php
require_once('WOVN.php/src/wovn_interceptor.php');
?>
<html>
<head>
<link rel="alternate" hreflang="en" href="http://ja.AAAAA.com/" />
</head>
<body>
test
</body>
</html>
