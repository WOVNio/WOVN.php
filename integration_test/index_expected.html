<html lang="en">
<head>
<script src="//j.wovn.io/1" async="true" data-wovnio="key=zwBmtA&amp;backend=true&amp;currentLang=ja&amp;defaultLang=ja&amp;urlPattern=path&amp;langCodeAliases={}&amp;langParamName=wovn&amp;version=0.0.1"> </script>
<link rel="alternate" hreflang="en" href="http://localhost/en/index.php">
<link rel="alternate" hreflang="ja" href="http://localhost/index.php">
</head>
<body>
test
</body>
</html>
